### DISCUSSION TITLE
Test Discussion

### DISCUSSION THREAD

**Discussion by bwateratmsft** [microsoft/vscode-docker#2707] [Q&A] [answered]
Hello world!

**Comment by bwateratmsft** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/36966225/107238100-7376c680-69f5-11eb-9522-5c73fbedc0eb.png)

**Comment by karolz-ms** [comment]
What is discussion? How is it different from an issue?

![image](https://user-images.githubusercontent.com/15271049/107253717-740a5f80-69eb-11eb-9dce-226bcb22ed0a.png)

**Reply by bwateratmsft** [reply]
It's sorta like a StackOverflow post. Better suited to Q&A than issues are. For example, we could convert #2694 to a discussion.

**Reply by karolz-ms** [reply]
That's neat 👍

**Reply by BigMorty** [reply]
![image](https://user-images.githubusercontent.com/947510/107674680-0dd04780-6c4c-11eb-89a1-2fcec953f722.png)


Let's give it a try and see how it goes!
